src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToVerifyNuGetReferenceCompat.cs

```diff
@@ -188,11 +189,11 @@ public void It_is_possible_to_disable_netfx_implicit_asset_target_fallback()
         [WindowsOnlyFact]
         public void It_chooses_lowest_netfx_in_default_atf()
         {
-            var testProjectName = "netcoreapp30_multiple_atf";
+            var testProjectName = $"{Regex.Replace(ToolsetInfo.CurrentTargetFramework, @"\.", "")}_multiple_atf";
```

Comment:
FYI @v-wuzhai We don't need to use Regex for this, a simple `String.Replace` would be fine if you need to remove the periods.